config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'users/groups'
+  get 'users/operations'
+  devise_for :users
+  resources :users, only: [ :index, :show] do
+    resources :groups, only: [ :index, :show, :new, :create] do
+      resources :operations, only: [ :index, :show, :new, :create]
+    end
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :users do
+    root :to => "home#index"
+  end
+  
+  root to: "home#index"
```

Comment:
- [ ] When I run your app locally it gives the following error but I was able to review your app with the deployed version so please try to fix it 
<img width="1430" alt="Screen Shot 2022-10-13 at 16 01 24" src="https://user-images.githubusercontent.com/61790587/195636517-fa158390-a3f9-4ad2-9f6a-41fd1380706d.png">